### PR TITLE
[Docs] Refactor docs

### PR DESCRIPTION
**Pull request by gaotongxiao**
## Update (Aug 23)
Major refactorization is done. Additional contents will be left to the next PR.

TOC for new docs, subject to change. Any new commits in this PR should be reviewed on the fly.

### Getting Started
- [x] Installation
- [x] Quick Start
- [x] Trainig
- [x] Testing
- [x] Inference
- [x] Deployment (Experimental)

### UNNAMED
- [x] Model Zoo
- [x] Dataset Zoo

### Customization
- [ ] Configuration System
  - [ ] ~~Configuration Description~~
  - [ ] ~~Weight initialization (init_cfg)~~
  - [x] Dataset types
  - [ ] ~~Available pipelines~~
- [ ] ~~Tutorials~~
  - [ ] ~~Dataset Preparation~~
  - [ ] ~~Dataset Customization~~
  - [ ] ~~Pipeline Customization~~
 
### Useful Tools
- [ ] ~~Dataset Converter~~
- [x] Utils

### Notes
- [ ] ~~Compatibility and migration guide~~
- [x] Changelog

### CODE REVIEW

**Comment by innerlee** [comment]
The corresponding pr could be appended to the items

File: docs/test.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Test
+
+### Test a Dataset
```
Comment (innerlee) [inline docs/test.md:3]: should be second level header?

File: docs/test.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+You can use the `tools/test.py` to perform single GPU inference. For example, to evaluate DBNet on IC15: (You can download pretrained models from [Model Zoo](modelzoo.md)):
+
+```shell
+./tools/dist_test.sh configs/textdet/dbnet/dbnet_r18_fpnc_1200e_icdar2015.py dbnet_r18_fpnc_sbn_1200e_icdar2015_20210329-ba3ab597.pth --eval hmean-iou
+```
+
+And here is the full usage of the script:
+
+```
```
Comment (innerlee) [inline docs/test.md:16]: ` ```shell `

File: docs/test.md (hunk 1 above)
Comment (innerlee) [reply docs/test.md:3]: Will there be another section on "test an image"?

Comment (gaotongxiao) [reply docs/test.md:3]: > Will there be another section on "test an image"?

Yes, but it will be added to the inference section.

**Pull request merged**